qualtricssurvey/qualtricssurvey.py

```diff
@@ -103,11 +115,11 @@ def get_resource_url(self, path):
         return resource_url
 
     def build_fragment(self,
-        html_source=None,
-        path_css=None,
-        path_js=None,
-        fragment_js=None,
-    ):
+                       html_source=None,
+                       path_css=None,
+                       path_js=None,
+                       fragment_js=None,
+                       ):
```

Comment:
Ya, drop `self` down a line and then unindent the block by 1. The closing comma should align w/ the `d` in `def`.